R/esqlabs-plot-export-configuration.R

```diff
@@ -1,14 +1,5 @@
 #' @title ExportConfiguration
 #' @description R6 class defining properties for saving a `ggplot` object
-#' @field name character defining the name of the file to be saved (without extension)
```

Comment:
Why remove?

Reply:
Because these are not arguments for initialize method.

Reply:
They are, the `initialize` method is just defined in `TLF`. I copied the documentation here because otherwise Roxygen complained. But if it works without, fine with me (even better so, I didn't like this duplication)